src/itest/kotlin/nl/lifely/zac/itest/ZakenRESTServiceTest.kt

```diff
@@ -436,137 +434,4 @@ class ZakenRESTServiceTest : BehaviorSpec({
             }
         }
     }
-//    Given("A zaak that is ontvankelijk") {
```

Comment:
Is this commented out to disable the tests or is it leftover from debugging

Reply:
Ah good call. Removed it now. This was your original test code which I moved to a separate test. Should be gone now.